server/src/routes/auth.signin.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const authController = require("../controllers/auth.controller");
+const{ validateSignupRequest, isRequestValidated, validateSigninRequest} = require('../validators/auth.validator')
+const router = express.Router();
+
+/** @route      GET /api/signup
+ *  @desc       fetch single signup
+ *  @access     Public
+ */
+
+router.post('/',validateSigninRequest, isRequestValidated, authController.signin);
+
+module.exports = router;
```

Comment:
Kindly don't create separate files for each of the route, as it is also an auth route so keep it in auth.route.js only and then import it to root.js